tests/integration/requirements.txt

```diff
@@ -2,6 +2,7 @@ boto3
 botocore
 docker
 juju==3.0.4
+kubernetes
```

Comment:
k8s compatibility is only guaranteed between one minor version and the next or previous. I would surely pin this dependency to 25.3.0 as in nginx-ingress-integrator charm to make sure we are not attempting to use features not present in our clusters

Reply:
Fixed